doc/src/modules/vector/api/classes.rst

```diff
@@ -2,13 +2,13 @@
 Essential Classes in sympy.vector (doctrings)
 =============================================
 
-CoordSysCartesian
-=================
+CoordSystem3D
```

Comment:
Keep in mind that if you change the public API we have to go through deprecation cycles. If this module is in 0.7.6, that will be the case. See https://github.com/sympy/sympy/wiki/Deprecating-policy